default/python/universe_generation/monsters.py

```diff
@@ -1,8 +1,82 @@
 import random
 import freeorion as fo
 import util
+from util import MapGenerationError
```

Comment:
Hm, if we import `MapGenerationError` that way, it's probably more consistent if we import the rest of the stuff from `util` the same way.